HEC.FDA.ViewModel/Hydraulics/GriddedData/GriddedImporterVM.cs

```diff
@@ -139,11 +158,12 @@ public void FolderSelected(string fullpath)
                         importResult.AddErrorMessage(result.ErrorMessage);
                     }
                 }
-                double prob = 0;
                 foreach (string dir in validDirectories)
                 {
-                    prob += .1;
-                    AddRow(Path.GetFileName(dir), Path.GetFullPath(dir), prob);
+                    string name = Path.GetFileName(dir);
+                    string[] tifFiles = Directory.GetFiles(dir, "*.tif");
+                    string tif = tifFiles[0]; //there should only be one, but if there are more, we'll just take the first one
```

Comment:
This enforces the Path property of a row actually being the full path to the tif. That's consistent with the other importers.